mathesar/signals.py

```diff
@@ -27,3 +28,11 @@ def create_table_settings(**kwargs):
 def compute_preview_column_settings(**kwargs):
     instance = kwargs['instance']
     _compute_preview_template(instance.table)
+
+
+@receiver(post_save, sender=DatabaseRole)
+def give_manager_access_to_public_schema(**kwargs):
+    if kwargs['created']:
+        instance = kwargs['instance']
+        public_schema = Schema.objects.get(oid=2200, database=instance.database)
```

Comment:
is the public schema always `2200`? can we put this in a variable so that it's clearer (e.g. `PUBLIC_SCHEMA_OID = 2200`) and doesn't seem like a random number?